packages/eslint-config-norton/rules/bestPractices.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Let's just put these rules directly in the `typescript-base.js` file rather than maintaining them separately. This file's location and name (`rules/bestPractices`) give the impression that it contains general best practices for ECMAScript, when they actually only apply to TypeScript. If you ever end up adding more custom "best practices," it might make sense to split things out again.